torch/__init__.py

```diff
@@ -1435,3 +1435,9 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
                   'use torch.sparse_coo_tensor(..., check_invariants=False) instead.')
     kwargs['check_invariants'] = False
     return torch.sparse_coo_tensor(*args, **kwargs)
+
+
+# dynamic registration of sparse triton kernels
+from torch.sparse import _register_impls
+lib = torch.library.Library("aten", "IMPL")
+_register_impls(lib)
```

Comment:
Alternatively, this could be moved into `torch/sparse/__init__.py` and/or `torch.nn.__init__.py`.